types/custom-functions-runtime/index.d.ts

```diff
@@ -9,33 +9,32 @@ office-js
 Copyright (c) Microsoft Corporation
 */
 
-////////////////////////////////////////////////////////////////
-//////////////////// Begin custom-functions-runtime ////////////
-////////////////////////////////////////////////////////////////
-
 /**
- * Enables you to map your own name that uses lowercase letters to a function.
+ * Specific to Excel Custom Functions. Enables you to set key-value pairs where the key is the uppercase id of a function in the custom function's JSON metadata file and the value is the name of the function as defined in the function's JavaScript file.
```

Comment:
Since it matches the [public documentation](https://docs.microsoft.com/en-us/office/dev/add-ins/excel/custom-functions-overview), I'm assuming the change from "lowercase" to "uppercase" was intentional.  However, it looks like the test uses a camelCase key ("addTen") - should it be updated as well?

Reply:
Good catch, thank you :)